.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "8"
+  - "9"
```

Comment:
like Simon said, just 8 should be good here

Reply:
Done